lib/smart_answer/calculators/arrested_abroad.rb

```diff
@@ -1,14 +1,12 @@
 module SmartAnswer::Calculators
   class ArrestedAbroad
     # created for the help-if-you-are-arrested-abroad calculator
-    attr_reader :data
+    attr_accessor :country
 
-    def initialize
-      @data = self.class.prisoner_packs
-    end
+    PRISONER_PACKS = YAML.load_file(Rails.root.join("config/smart_answers/prisoner_packs.yml")).freeze
 
     def generate_url_for_download(country, field, text)
-      country_data = @data.find { |c| c["slug"] == country }
+      country_data = PRISONER_PACKS.find { |c| c["slug"] == country }
```

Comment:
I think `country_data` should be a separate method. That way it can be reused (for example in `get_country_regions`), and would also ensure a consistent way of identifying the current country data from the country slug.

Reply:
yup - done the suggested change!